post-scan-actions/aws-python-promote-or-quarantine/template.yml

```diff
@@ -0,0 +1,124 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: cloudone-filestorage-plugin-action-promote-or-quarantine
+    Description: >-
+      After a scan occurs, this example application places clean files in one bucket
+      and malicious files in another.
```

Comment:
Hi @dougc-trend ,
In case you are not aware, descriptions in `Metadata` and `Parameters` are also changeable.

Reply:
I'm sorry, I don't understand what you are trying to tell me.

Reply:
I mean the descriptions in the `template.yml` are also under your command. You can check the screenshot I took as the result of those descriptions showing on AWS console. I wasn't sure that whether you have reviewed those descriptions and everything looks good to you or you didn't know those are also reviewable.